helios-services/src/main/java/com/spotify/helios/servicescommon/ServiceParser.java

```diff
@@ -185,6 +186,11 @@ public ServiceParser(final String programName, final String description, final S
         .setDefault(new ArrayList<String>())
         .help("Kafka brokers to bootstrap with");
 
+    pubsubTopicPrefixArg = parser.addArgument("--pubsub-topic-prefix")
+        .action(append())
+        .setDefault(new ArrayList<String>())
+        .help("Google pubsub topic prefixes to send to");
```

Comment:
Yes, that would be the idea. I followed the current style of not documenting the topics… ;)

Is there a good place to put it? A new `.md` file in `docs/`?